Makefile

```diff
@@ -0,0 +1,50 @@
+.ONESHELL:
+SHELL = bash
+
+GIT_USER_ID = libretime
+GIT_REPO_ID = api-client
+
+# https://github.com/OpenAPITools/openapi-generator/releases
+GENERATOR_VERSION = v6.0.0-beta
+GENERATOR_CLI = docker run \
+	--rm \
+	--volume="$$PWD:/local" \
+	--user="$$UID:$$GID" \
+	openapitools/openapi-generator-cli:$(GENERATOR_VERSION)
+
+GENERATATE_CMD = $(GENERATOR_CLI) generate \
+		--input-spec /local/schema.yml \
+		--git-user-id $(GIT_USER_ID) \
+		--git-repo-id $(GIT_REPO_ID)
+
+all: javascript-client php-client python-client
+
+.PHONY: javascript-client
+javascript-client:
+	$(GENERATATE_CMD) \
+		--generator-name typescript-axios \
+		--output /local/javascript-client \
+		--package-name libretime-client \
+		--additional-properties=npmName=libretime-client \
+		--additional-properties=supportsES6=true \
```

Comment:
I think we should compile 2 different targets, both es5 and es6 so we have a dist for older browser, and use the es6 dist for the webapp (vue) that will recompile the library anyway.